docs/meshstack.azure.index.md

```diff
@@ -54,136 +52,38 @@ It is important that the provided euid's are **case-sensitive** and must match t
-            }
-          }
-        ],
-        "hasReadAccess": false
-      },
-      "type": "Microsoft.Billing/billingAccounts"
-    }
-  ]
-}
-```
-
-The value for a billing scope and id are the same thing. The id for your enrollment account is the billing scope under which the subscription request is initiated. Please note this ID down as it needs to be used as the `enrollmentAccountId` in the [DHALL provisioning configuration](meshstack.azure.config.md#provisioning-configuration).
-
-#### Ensuring Retained Subscription Owners
-
-Azure requires that there's at least one "Owner" or "Classic Administrator" role assignment on each Subscription. Unfortunately, it's not a sufficient workaround to inherit the Owner role via the Management Group Hierarchy onto the Subscription. Instead a direct role assignment must exist. This owner can also be the Azure [Blueprint Service Principal](#blueprint-configuration)
-
-In contrast to other provisioning methods, EA provisioning will not retain a default "Classic Administrator" role assignment on the subscription from the billing account owner. Operators should therefore configure at least one explicit owner under `subscriptionOwnerObjectIds`. This owner can also be an empty AAD group or Service Principal.
-
-> You should never grant subscription owner roles to the meshStack replicator SPN.
-
-### Pre-provisioned Subscriptions
-
-If your organization does not have access to an Enterprise Enrollment, you can alternatively configure meshcloud to
-consume subscriptions from a pool of externally-provisioned subscriptions. This is useful for smaller organizations that whish
-to use "Pay-as-you-go" subscriptions or if you're organization partners with an [Azure Cloud Solution Provider](https://docs.microsoft.com/en-us/azure/cloud-solution-provider/overview/azure-csp-overview) to provide your subscriptions.
-
-The meshcloud Azure [replication](./meshcloud.tenant.md) detects externally-provisioned subscriptions based on a configurable prefix in the subscription
-name. Upon assignment to a meshProject, the subscription is inflated with the right [Landing Zone](./meshstack.azure.landing-zones.md) configuration
-and removed from the subscription pool.
-
 ## Service Principal Setup
 
-meshStack uses separate service principals for different tasks following the best practice of least privilege principle.
-
-### Replicator
+meshStack uses separate service principals for different tasks following the best practice of least privilege principle. Depending on the way how you want to setup Subscription creation (there are two possible ways: Pre-provisioned Subscriptions or using an Enterprise Enrollment Account), you can use either an App Registration or an Enterprise Application. We recommend just using an Enterprise Application to keep things consistent and which is **required** to use the Enterprise Agreement Account based [Subscription provisioning method](#enterprise-enrollment-account).
```

Comment:
q: I am still unsure about the relation App Registration <-> Enterprise Application. If I understood [this](https://www.seb8iaan.com/the-difference-between-azuread-app-registrations-and-enterprise-applications-explained/) correctly, an Enterprise Application is simply an instance of an App Registration. So the formulation here might be confusing to some Operators. 

Probably someone with more Azure experience like @SimonTheLeg or @younGihan could proof-read this specific section.

Reply:
Sounds right for the current context but there are also some other differences

Reply:
Out of curiosity: What are other differences?

Reply:
hm yeah I wanted to keep the terminologic of the buttons you also find on the Azure portal panel. But maybe I should emphasis more that probably the best idea is to just use an Enterprise Application (which yea, is also an App Registration as well :D)